mobile/src/app/app.tsx

```diff
@@ -5,15 +5,14 @@ import React, { type FC } from 'react';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
 import { Provider as StoreProvider } from 'react-redux';
 
+import { globalStyles } from '~/bundles/common/styles/global-styles';
```

Comment:
Let's create styles.ts in styles folder and re-export global styles 
```suggestion
import { globalStyles } from '~/bundles/common/styles';
```